primitives/state-machine/src/testing.rs

```diff
@@ -119,6 +126,16 @@ impl<H: Hasher, N: ChangesTrieBlockNumber> TestExternalities<H, N>
 		}
 	}
 
+	/// Apply the changes made by the offchain indexing API.
+	pub fn sync_offchain_index_changes(&mut self) {
+		self.offchain_db.apply_offchain_changes(&mut self.offchain_overlay);
+	}
+
+	/// Create a sync wrapper around the persistent part of the offchain worker storage.
```

Comment:
"a sync wrapper"? This method just returns a shared reference, no?

Reply:
sync as in rust trait `Sync`, not as in not-async